packages/voter-dapp/src/ActiveRequests.js

```diff
@@ -353,7 +363,8 @@ function ActiveRequests({ votingAccount, votingGateway, snapshotContract }) {
           identifier: pendingRequests[index].identifier,
           time: pendingRequests[index].time,
           price: decryptedCommits[index].price.toString(),
-          salt: decryptedCommits[index].salt
+          salt: decryptedCommits[index].salt,
+          ancillaryData: "0x"
```

Comment:
Shouldn't we be reading the ancillary data from `decryptedCommits` or is the decryption not working?

Reply:
Actually, see my comment below in the `commits.push` code.

Reply:
you might be right about this, probably a mistake.